apps/graphql/src/apps/mmb/dataloaders/ConfirmPayment.js

```diff
@@ -8,7 +8,7 @@ import type {
   ConfirmPaymentPayloadType,
   ConfirmPaymentOutputType,
   ConfirmPaymentAPIResponse,
-} from '../Booking';
+} from '../mmb';
```

Comment:
If these are being used within `booking` and `manageMyBooking`, we have a folder for common types also.